src/app/logging/page.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+import { BasicRoundedButton } from "@/components/buttons/basic-rounded-button/Basic-rounded-button";
+import { useAuthSession } from "@/lib/contexts/auth-context/auth-context";
+import { Link } from "@mui/material";
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import moduleStyles from "../home-page.module.css";
+
+export default function Logging() {
+  // State to hold the current date
+  const [currentDate, setCurrentDate] = useState("");
+
+  // useEffect to set the current date when the component mounts
+  useEffect(() => {
```

Comment:
Similar comment here - we can set the current date when you declare the state variable.

Reply:
Same comment as the one above